### DISCUSSION TITLE
Samples documentation possibly out of date

### DISCUSSION THREAD

**Discussion by alzcurda** [eclipse-edc/Samples#220] [Q&A]
We are using the Samples code and we see that some json contains values ​​that do not correspond to the API documentation. For example:

https://github.com/eclipse-edc/Samples/blob/main/transfer/transfer-00-prerequisites/resources/dataplane/register-data-plane-provider.json

Its content is the following:

```
{
  "@context": {
    "@vocab": "https://w3id.org/edc/v0.0.1/ns/"
  },
  "@id": "http-pull-provider-dataplane",
  "url": "http://localhost:19192/control/transfer",
  "allowedSourceTypes": [
    "HttpData"
  ],
  "allowedDestTypes": [
    "HttpProxy",
    "HttpData"
  ],
  "properties": {
    "https://w3id.org/edc/v0.0.1/ns/publicApiUrl": "http://localhost:19291/public/"
  }
}
```

But if I go to https://app.swaggerhub.com/apis/eclipse-edc-bot/management-api/0.5.1#/Dataplane%20Selector/addEntry and select the content of [/v2/dataplanes] I found:

```
  post:
      tags:
      - Dataplane Selector
      description: Adds one dataplane instance to the internal database of the selector
      operationId: addEntry
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/DataPlaneInstanceSchema'
      responses:
        "200":
          description: Entry was added successfully to the database
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/IdResponse'
        "400":
          description: Request body was malformed
          content:
            application/json:
              schema:
                type: array
                example: null
                items:
                  $ref: '#/components/schemas/ApiErrorDetail'
```

And the detail of  '#/components/schemas/DataPlaneInstanceSchema' is:

```
 DataPlaneInstanceSchema:
      required:
      - '@context'
      - allowedDestTypes
      - allowedSourceTypes
      - url
      type: object
      properties:
        '@context':
          type: object
          example: null
        '@id':
          type: string
          example: null
        '@type':
          type: string
          example: https://w3id.org/edc/v0.0.1/ns/DataPlaneInstance
        allowedDestTypes:
          uniqueItems: true
          type: array
          example: null
          items:
            type: string
            example: null
        allowedSourceTypes:
          uniqueItems: true
          type: array
          example: null
          items:
            type: string
            example: null
        lastActive:
          type: integer
          format: int64
          example: null
        turnCount:
          type: integer
          format: int32
          example: null
        url:
          type: string
          format: url
          example: null
      example:
        '@context':
          '@vocab': https://w3id.org/edc/v0.0.1/ns/
        '@id': your-dataplane-id
        url: http://somewhere.com:1234/api/v1
        allowedSourceTypes:
        - source-type1
        - source-type2
        allowedDestTypes:
        - your-dest-type
```

In short, in the json we have the ta properties, but they do not appear in RESTAPI. Is this parameter necessary?

I am trying to access to the ontology "https://w3id.org/edc/v0.0.1/ns/" but I only get a "The requested URL was not found on this server.". Is there any documentation regarding this that I can access?

Thanks.
